EnglishFilter/sections/antiadblock.txt

```diff
@@ -510,11 +510,11 @@ freegogpcgames.com,omgpu.com,arcadepunks.com#%#//scriptlet("set-constant", "DHAn
 @@/wp-content/plugins/ad-ace//includes/adblock-detector/advertisement.js$~third-party
 !
 ! AdUnblocker
-graphicux.com,w3layouts.com,musike.net,windowsmatters.com,sonixgvn.net,freeiptvlists.com,avoiderrors.com,gatcha.org,sayrodigital.com,justandroid.net,4horlover.com,topnewsshow.com,3dzip.org,ovagames.com#%#//scriptlet('prevent-addEventListener', 'load', 'daadb_get_data')
-graphicux.com,w3layouts.com,musike.net,windowsmatters.com,sonixgvn.net,freeiptvlists.com,avoiderrors.com,gatcha.org,sayrodigital.com,justandroid.net,4horlover.com,topnewsshow.com,3dzip.org,ovagames.com#%#//scriptlet("set-constant", "adsbygoogle.length", "undefined")
-||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$script,xmlhttprequest,redirect=googlesyndication-adsbygoogle,domain=graphicux.com|w3layouts.com|musike.net|windowsmatters.com|sonixgvn.net|freeiptvlists.com|avoiderrors.com|gatcha.org|sayrodigital.com|justandroid.net|4horlover.com|topnewsshow.com|3dzip.org|ovagames.com
+kisahdunia.com,graphicux.com,w3layouts.com,musike.net,windowsmatters.com,sonixgvn.net,freeiptvlists.com,avoiderrors.com,gatcha.org,sayrodigital.com,justandroid.net,4horlover.com,topnewsshow.com,3dzip.org,ovagames.com#%#//scriptlet('prevent-addEventListener', 'load', 'daadb_get_data')
+kisahdunia.com,graphicux.com,w3layouts.com,musike.net,windowsmatters.com,sonixgvn.net,freeiptvlists.com,avoiderrors.com,gatcha.org,sayrodigital.com,justandroid.net,4horlover.com,topnewsshow.com,3dzip.org,ovagames.com#%#//scriptlet("set-constant", "adsbygoogle.length", "undefined")
```

Comment:
I don't find those scriptlets useful. 
1. Remove `kisahdunia.com` from common rules.
2. Add rules below with task link in this section.
```
! https://github.com/AdguardTeam/AdguardFilters/issues/72828
||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$important,script,xmlhttprequest,redirect=googlesyndication-adsbygoogle,domain=kisahdunia.com
@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=kisahdunia.com
```

Reply:
Where should I exactly put the rules? There is a common section for the rules you suggested.

Reply:
It's a little lower after `! END: Ads replacement`.

Reply:
Now?

Reply:
I changed it a bit, it's ok now.